src/errors.rs

```diff
@@ -230,7 +234,7 @@ impl std::fmt::Display for Error {
     fn fmt(&self, fmt: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
         match self {
             Error::MeiliSearchError {
-                message,
+                error_message,
                 error_code,
                 error_type,
                 error_link,
```

Comment:
Do you prefer to keep this naming for clarity purposes?

Reply:
Yes, plus this is name the same in the MeiliSearch code base, and this SDK code base is inspired of the MeiliSearch one, that's why I did not change this :)